pkg/controller/service/ingress_reconciler_test.go

```diff
@@ -35,6 +35,8 @@ func TestIngressReconcilerIntegration(t *testing.T) {
 	svc := &corev1.Service{}
 	svc.UID = "test-svc-uid"
 	svc.Name = "test-service"
+	svc.APIVersion = "v1"
+	svc.Kind = "Service"
```

Comment:
for my understanding: what is the use of adding apiversion and kind here ?

Reply:
It's just a difference in how the fake client works due to the version bump.